tests/interestTest.php

```diff
@@ -17,6 +17,29 @@ class interestsTests extends TestCase {
 	 * Make sure unit tests are running.
 	 */
 	public function testNamespaceLoaded() {
-		$this->assertTrue( function_exists( '\\Pantheon\\EI\\WP\\Interest\\bootstrap' ) );
+		$this->assertTrue(
+			function_exists( '\\Pantheon\\EI\\WP\\Interest\\bootstrap' ),
+			'bootstrap function does not exist'
+		);
+	}
+
+	/**
+	 * Test script registration.
+	 */
+	public function testRegisterScript() {
+		$this->assertTrue( 
+			function_exists( '\\Pantheon\\EI\\WP\\Interest\\register_script' ),
+			'register_script function does not exist'
+		);
+	}
+
+	/**
+	 * Test Interest localization.
+	 */
+	public function testInterestLocalization() {
+		$this->assertTrue( 
+			function_exists( '\\Pantheon\\EI\\WP\\Interest\\localize_interests' ),
```

Comment:
In a traditional WP Unit environment, we could test what happens when we're inside a loop and run this function (e.g. test the filters). We could still do that, although that sort of breaks the rule of "don't test WordPress doing WordPress things" and would likely be more helpful on the actual test site. 

I guess my point is, if it doesn't seem like we can test anything outside of the existence of specific functions, then the tests aren't really adding much value and should be removed in favor of E2E testing on a dev environment.